app/helpers/proxmox_vm_volumes_helper.rb

```diff
@@ -52,12 +52,14 @@ def parsed_typed_volumes(args, type, parsed_vm)
   end
 
   def parse_hard_disk_volume(args)
+    logger.debug(format(_('parse_hard_disk_volume(): args=%<args>s'), args: args))
```

Comment:
Generally speaking we don't translate debug statements. They only make it hard to figure out.